spec/models/like_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+
+feature 'Liking photos' do
+
+  context 'Signed in' do
+
+    let!(:sea) {Photo.create(title:'Awesome sea',
+                             image: File.new("spec/features/sea.jpg") )}
```

Comment:
Expression at 9, 72 should be on its own line.<br>Space inside parentheses detected.